src/peft/tuners/lora/model.py

```diff
@@ -269,11 +269,13 @@ def _replace_module(self, parent, child_name, new_module, child):
                 weight = (
                     child.qweight
                     if hasattr(child, "qweight")
-                    else child.W_q
-                    if hasattr(child, "W_q")
-                    else child.weight
-                    if hasattr(child, "weight")
-                    else next(child.parameters())
+                    else (
```

Comment:
Why this change? This can be reverted, right?

Reply:
You're right, this change is due to my formatter settings. I've reverted the change.